ocs_ci/utility/oc_utils.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from semantic_version import Version
+
+from ocs_ci.utility.utils import exec_cmd
+
+log = logging.getLogger(__name__)
+
+
+def get_client_version():
+    """
+    Get the major.minor version of the openshift client
+
+    Returns:
+        str: major.minor oc version
+
+    """
+    # get version info from oc
+    cmd = "oc version"
+    resp = exec_cmd(cmd, ignore_error=True)
```

Comment:
I believe we shouldn't ignore error, the resp might not be accurate for further processing.

Reply:
We are ignoring the error because `oc version` will return an error code if the server version is not returned properly:

```
$oc version
Client Version: 4.8.0-0.nightly-2021-03-22-104536
error: You must be logged in to the server (Unauthorized)
```

In this case, I still want to be able to parse out the client version so I am ignoring the error. We could do this differently though. We could except the `CommandFailed` and still parse the stdout if it exists but I think the result would be similar to passing `ignore_error=True`.

Reply:
I think its fine in that case

Reply:
How it will behave, if `oc` client doesn't exist in the time of execution of this function?

Reply:
@dahorak good point, I didn't actually test without a client present I will have to consider that and make sure we continue gracefully if it doesn't exist at all.